### DISCUSSION TITLE
I have question.

### DISCUSSION THREAD

**Discussion by paar13kr** [filecoin-project/lotus#8507] [General Q&A] [answered]
1. Does Protocol Labs have a mortgage service?
2. How can i extend sectors that are nearing expiration?
3. How does it work for block mining? By what mechanism is it compensated?

**Comment by rjan90** [comment, marked as answer]
Hey @paar13kr!

1. Protocol Labs does not have an internal mortgage service. But there are numerous ecosystem players that offer mortgage service for Filecoin Storage Provider: [Like this](https://blog.coinlist.co/introducing-low-collateral-filecoin-borrowing-to-grow-decentralized-storage/) and a couple other providers are listed in [this video](https://youtu.be/euVyc33ePFA?t=54).

2. You can extend sectors by [following this guide](https://lotus.filecoin.io/storage-providers/operate/sector-pledging/#extend-sectors).

3. You are eligible to win block-rewards after you have passed 10TiB in raw power - the amount of block rewards you get are dependent on how much storage power you have in the network. For every block one wins - 25% of the block reward is immediately available - the rest will be vested linearly over the next 180 days.
